src/PointwiseFunctions/GeneralRelativity/Tags.hpp

```diff
@@ -28,6 +28,11 @@ struct SpatialMetric : db::SimpleTag {
   static std::string name() noexcept { return "SpatialMetric"; }
 };
 template <size_t Dim, typename Frame, typename DataType>
+struct DetAndInverseSpatialMetric : db::SimpleTag {
```

Comment:
I have added tests of compute tag names in this and all other similar PRs. Since these tests call the parent simple tags' `name`'s, would these name tests count?

Reply:
From my perspective, one thing is to test the simple tag's name (as in making sure `name( )` works and the name itself is correctly spelled), and a different thing is that the compute item's `name()` function itself also works.

For consistency with what's already in develop, I'd add both. (Simple tag tests can go in a future PR, while compute items should be tested here.) Maybe in the future we come up with a less redundant way to test these.

Reply:
Opened #1493 for this